### PR TITLE
CASMSEC-376: Get upstream K8s policies running via kyverno in audit mode for Alice for fresh install and upgrade cases

### PR DESCRIPTION
**Pull request by pradeepkumargl**
CASMSEC-376:
Get upstream K8s policies running via kyverno in audit mode for Alice for fresh install and upgrade cases

Kyverno 1.6.0 upstream policies are uploaded into new repo.
Currently, baseline policies run in audit mode.

### CODE REVIEW

**Comment by pradeepkumargl** [comment]
Testing steps:
------------
Taken the code from PR and tested on mug

Following are the test logs:
-----------------------------
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 # helm install kyverno-policies charts/kyverno-policies -n kyverno
NAME: kyverno-policies
LAST DEPLOYED: Fri Jan 13 16:17:37 2023
NAMESPACE: kyverno
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing kyverno-policies v2.2.0 😀

We have installed the collection of policies that implement the various levels of Kubernetes Pod Security Standards.

Visit https://kyverno.io/policies/ to find more sample policies.
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 # kubectl get clusterpolicy
NAME                             BACKGROUND   ACTION   READY
disallow-capabilities            true         audit
disallow-host-namespaces         true         audit
disallow-host-path               true         audit    true
disallow-host-ports              true         audit    true
disallow-host-process            true         audit
disallow-privileged-containers   true         audit
disallow-proc-mount              true         audit
disallow-selinux                 true         audit
restrict-apparmor-profiles       true         audit
restrict-seccomp                 true         audit
restrict-sysctls                 true         audit
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 #
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 # kubectl get polr -A
NAMESPACE           NAME                        PASS   FAIL   WARN   ERROR   SKIP   AGE
argo                polr-ns-argo                20     2      0      0       0      116d
cdelatte            polr-ns-cdelatte            0      0      0      0       0      59d
ceph-cephfs         polr-ns-ceph-cephfs         4      7      0      0       0      119d
ceph-rbd            polr-ns-ceph-rbd            4      7      0      0       0      119d
cert-manager-init   polr-ns-cert-manager-init   0      0      0      0       0      119d
cert-manager        polr-ns-cert-manager        12     0      0      0       0      119d
default             polr-ns-default             4      1      0      0       0      111d
gatekeeper-system   polr-ns-gatekeeper-system   22     3      0      0       0      119d
hnc-system          polr-ns-hnc-system          4      0      0      0       0      119d
ims                 polr-ns-ims                 30     10     0      0       0      107d
istio-operator      polr-ns-istio-operator      4      0      0      0       0      114d
istio-system        polr-ns-istio-system        24     0      0      0       0      119d
kyverno             polr-ns-kyverno             4      0      0      0       0      114d
metallb-system      polr-ns-metallb-system      11     2      0      0       0      119d
nexus               polr-ns-nexus               8      1      0      0       0      119d
opa                 polr-ns-opa                 20     0      0      0       0      119d
operators           polr-ns-operators           41     9      0      0       0      119d
pki-operator        polr-ns-pki-operator        4      0      0      0       0      114d
services            polr-ns-services            1004   137    0      0       0      119d
sma                 polr-ns-sma                 186    6      0      0       0      114d
spire               polr-ns-spire               52     14     0      0       0      119d
sysmgmt-health      polr-ns-sysmgmt-health      331    10     0      0       0      119d
tapms-operator      polr-ns-tapms-operator      4      0      0      0       0      114d
user                polr-ns-user                4      2      0      0       0      119d
vault               polr-ns-vault               19     4      0      0       0      119d
velero              polr-ns-velero              8      3      0      0       0      119d
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 #
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 # helm delete -n kyverno kyverno-policies
release "kyverno-policies" uninstalled
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 # kubectl get clusterpolicy                            
No resources found
ncn-m001:/tmp/pradeep/Jan13/cray-kyverno-policies-upstream-CASMSEC-376 #

**Comment by jeremy-duckworth** [comment]
Why are we making a copy of this chart versus including it as a helm dependency, and then overriding values?

**Review by jeremy-duckworth — CHANGES_REQUESTED** [review]
Please use a helm chart dependency to pull in the upstream chart vs. maintaining separate copies of the content. Or, advise why you're not opting to do so. Thanks.

**Comment by pradeepkumargl** [comment]
> Why are we making a copy of this chart versus including it as a helm dependency, and then overriding values?

>> I have implemented changes to add dependency in helm and pull from upstream instead of storing locally.

**Comment by pradeepkumargl** [comment]
> Please use a helm chart dependency to pull in the upstream chart vs. maintaining separate copies of the content. Or, advise why you're not opting to do so. Thanks.

>> I have implemented changes to add dependency in helm and pull from upstream instead of storing locally.

File: charts/kyverno-policies/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dependencies:
+  - name: kyverno-policies
+    repository: https://kyverno.github.io/kyverno/
+    version: v2.3.2
+home: https://github.com/Cray-HPE/cray-kyverno-policies-upstream
+sources:
+  - https://github.com/kyverno/policies
+maintainers:
+  - name: pradeepg
+    url: https://kyverno.io/
```
Comment (jeremy-duckworth) [inline charts/kyverno-policies/Chart.yaml:22]: You are not the maintainer of the upstream content. If you are going to include this, use the GitHub URL (Cray/HPE) to the repo.

**Review by jeremy-duckworth — CHANGES_REQUESTED** [review]

**Review by jeremy-duckworth — APPROVED** [review]

**Review by bklei — APPROVED** [review]

**Review by kburns-hpe — APPROVED** [review]

**Pull request merged**